tools/Google.Cloud.Tools.ReleaseManager/GenerateProjectsCommand.cs

```diff
@@ -180,22 +180,28 @@ public static void RewriteDocsRootIndex(ApiCatalog catalog)
 
         /// <summary>
         /// Updates the dependencies in an API for known packages, but only if the default
-        /// version is later than the current one.
+        /// version is later than the current one, with the same major version number.
         /// </summary>
-        public static void UpdateDependencies(ApiMetadata api)
+        public static void UpdateDependencies(ApiCatalog catalog, ApiMetadata api)
         {
             UpdateDependencyDictionary(api.Dependencies, "dependencies");
             UpdateDependencyDictionary(api.TestDependencies, "testDependencies");
 
             void UpdateDependencyDictionary(SortedDictionary<string, string> dependencies, string jsonName)
             {
+                // We want to update any dependencies to "external" packages as listed in DefaultPackageVersions,
+                // but also "internal" packages such as Google.LongRunning.
+                Dictionary<string, string> allDefaultPackageVersions = DefaultPackageVersions
```

Comment:
nit: Maybe do this after the if that follows, so if there are no dependencies this is not done.

Reply:
Great minds think alike - already in the amended commit :)